atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -503,18 +505,22 @@ private void loadWithTs(TableReference tableRef,
 
         if (log.isTraceEnabled()) {
             log.trace("Loading {} cells from {} {}starting at timestamp {}, partitioned across {} nodes.",
-                    cells.size(), tableRef, loadAllTs ? "for all timestamps " : "", startTs, totalPartitions);
+                    SafeArg.of("cells", cells.size()),
+                    UnsafeArg.of("table", tableRef),
+                    SafeArg.of("timestampClause", loadAllTs ? "for all timestamps " : ""),
+                    SafeArg.of("startTs", startTs),
+                    SafeArg.of("totalPartitions", totalPartitions));
         }
 
         List<Callable<Void>> tasks = Lists.newArrayList();
         for (Map.Entry<InetSocketAddress, List<Cell>> hostAndCells : hostsAndCells.entrySet()) {
             if (log.isTraceEnabled()) {
                 log.trace("Requesting {} cells from {} {}starting at timestamp {} on {}",
-                        hostsAndCells.values().size(),
-                        tableRef,
-                        loadAllTs ? "for all timestamps " : "",
-                        startTs,
-                        hostAndCells.getKey());
+                        SafeArg.of("cells", hostsAndCells.values().size()),
+                        UnsafeArg.of("table", tableRef),
+                        SafeArg.of("timestampClause", loadAllTs ? "for all timestamps " : ""),
+                        SafeArg.of("startTs", startTs),
+                        SafeArg.of("ipPort", hostAndCells.getKey()));
```

Comment:
because service hostnames are part of the logging envelope, they are de facto safe already.  see link I sent you